### ISSUE TITLE
Instagram Feeds Don't Work

### ISSUE THREAD

**Issue by yakovv8** [nt1m/livemarks#469]
**Feed URL**
Any Instagram profile link.

**Add-on version**
3.6

**Describe the bug**
Instagram links work properly on a different feed reader but the folders on Livemarks remain empty and don't update (YouTube feeds work fine).

**To Reproduce**
Create a new feed with an Instagram profile link as the URL.

**Expected behavior**
I expected new posts to show up as bookmarks.

**Comment by nt1m** [comment]
Instagram doesn't seem to use traditional RSS feeds from what I'm seeing. Do you know which RSS readers work with Instagram? I'm fairly sure they'll use some custom made solution for instagram, but I can double check to be sure

**Comment by yakovv8** [comment]
Feedbro works with Instagram. https://addons.mozilla.org/en-US/firefox/addon/feedbroreader/?utm_content=search&utm_medium=referral&utm_source=addons.mozilla.org

**Comment by nt1m** [comment]
I actually tried feedbro and all the instagram profiles I tried to add were blank. What steps are you using to add the instagram feed?

**Comment by yakovv8** [comment]
Yeah that happens for me too, but the feed will eventually update within a day or two (not sure why). I was wondering if this was also the case for Livemarks or if the functionality is not built in at all. I usually add feeds on Feedbro through the "find feeds in current tab" option from the extension icon.

**Comment by nt1m** [comment]
I quickly looked at Feedbro's source code and they definitely have special code for instagram that hooks into instagram's APIs. Instagram doesn't actually have RSS feeds.

**Comment by yakovv8** [comment]
Ah I see, thanks. I would love if Livemarks had that functionality.
